_data/locales/en.yml

```diff
@@ -2,6 +2,7 @@ en:
   locale_name: English
   subtitle: The missing package manager for macOS (or Linux)
   pagecontent:
+    search: Search
```

Comment:
can you add this to all `locales`? That'll allow removal of `if t.pagecontent.search` in `base.html`

Reply:
@MikeMcQuaid done 🙂